crates/nu-command/src/filesystem/save.rs

```diff
@@ -121,9 +121,11 @@ impl Command for Save {
                             } else {
                                 match stderr {
                                     ChildPipe::Pipe(mut pipe) => {
-                                        io::copy(&mut pipe, &mut io::sink())
+                                        io::copy(&mut pipe, &mut io::stderr())
+                                    }
+                                    ChildPipe::Tee(mut tee) => {
+                                        io::copy(&mut tee, &mut io::stderr())
```

Comment:
I'm not sure about using `stack.stderr()`.  Because it's an indicator for the symbol after `save` command.
For the example:
```nushell
nu --testbin echo_env_stderr PATH e>| save -f /tmp/aa e> "/tmp/bb"
```
The `stack.stderr()` is a file, but I think the result should be saved in `/tmp/aa`

Reply:
I'm not sure about using `stack.stderr()`.  Because it's an indicator for the symbol after `save` command.
For the example:
```nushell
nu --testbin echo_env_stderr PATH e>| save -f /tmp/aa e> "/tmp/bb"
```
The `stack.stderr()` is a `OutDest::File`, but I don't think the stderr output of `nu --testbin echo_env_stderr PATH` should not be saved in `/tmp/bb`.

Reply:
Your first example saves the output to `/tmp/aa` when using `stack.stderr()`.
```nushell
> nu --testbin echo_env_stderr PATH e>| save -f /tmp/aa e> /tmp/bb

> open /tmp/aa
/home/iama/.local/bin:/home/iama/.local/share/cargo/bin:/home/iama/.local/share/pnpm:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/usr/lib/rustup/bin

> open /tmp/bb

```

Reply:
Oh, sorry, I wanted to type `| save` rather than `e>| save` in my first example

Reply:
> Maybe we can use `io::stderr()` first, and then deprecate `save --stderr` in your new pr?

Yep, that works!